File: Seção 13/App1_NossoChat/App1_NossoChat/App1_NossoChat/Model/Mensagem.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace App1_NossoChat.Model
{
    public class Mensagem
    {
        public int id { get; set; }
        public int id_chat { get; set; }
        public int id_usuario { get; set; }
        public string mensagem { get; set; }
        public Usuario usuario { get; set; }
    }
}

File: webservices-rest/application/app/Mensagem.php
<?php

namespace App;

use Illuminate\Database\Eloquent\Model;

class Mensagem extends Model {
    protected $table = 'ws_xf2018_messagem';
    protected $fillable = ['mensagem'];
    
    
    public function usuario() {
        return $this->belongsTo('App\Usuario', 'id_usuario');
    }
    /*
    public function chat(){
        return $this->belongsTo('App\Chat', 'id_chat', 'id');
    }*/
}